funnel/templates/notifications/layout_email.html.jinja2

```diff
@@ -1,45 +1,511 @@
 <!DOCTYPE html>
-<html lang="{{ get_locale() }}">
-  <head>
+<html lang="{{ get_locale() }}" xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
+<head>
+    <meta charset="utf-8"> {# utf-8 works for most cases #}
+    <meta name="viewport" content="width=device-width"> {# Forcing initial-scale shouldn't be necessary #}
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"> {# Use the latest (edge) version of IE rendering engine #}
+    <meta name="x-apple-disable-message-reformatting">  {# Disable auto-scale in iOS 10 Mail entirely #}
+    <meta name="format-detection" content="telephone=no,address=no,email=no,date=no,url=no"> {# Tell iOS not to automatically link certain text strings. #}
+    <meta name="color-scheme" content="light">
+    <meta name="supported-color-schemes" content="light">
+    <title></title> {#   The title tag shows in email notifications, like Android 4.4. #}
+
+    {# What it does: Makes background images in 72ppi Outlook render at correct size. #}
+    <!--[if gte mso 9]>
+    <xml>
+        <o:OfficeDocumentSettings>
+            <o:AllowPNG/>
+            <o:PixelsPerInch>96</o:PixelsPerInch>
+        </o:OfficeDocumentSettings>
+    </xml>
+    <![endif]-->
+
+    {# Outlook / @font-face : BEGIN #}
+
+    {# Desktop Outlook chokes on web font references and defaults to Times New Roman, so we force a safe fallback font. #}
+    <!--[if mso]>
+        <style>
+            * {
+                font-family: sans-serif !important;
+            }
+        </style>
+    <![endif]-->
+
+    {# Outlook / @font-face : END #}
+
     {% block stylesheet -%}
-      <style type="text/css">
-        .btn, a.btn:link, a.btn:visited {
-          background-color: #428bca;
-          border-radius: 3px;
-          border: 1px solid #357ebd;
-          color: white;
-          margin-bottom: 6px;
-          padding: 6px 12px;
-          text-decoration: none;
-          white-space: nowrap;
-        }
-        .btn-bar {
-          margin: 2em 0;
-        }
-        blockquote {
-          margin-left: 0;
-          padding-left: 1em;
-          border-left: 3px solid #ccc;
-        }
-      </style>
+    {# CSS Reset : BEGIN #}
+    <style>
+
+        /* What it does: Tells the email client that only light styles are provided but the client can transform them to dark. A duplicate of meta color-scheme meta tag above. */
```

Comment:
These should be converted into Jinja2 comments so they're excluded from the rendered email.

Reply:
The CSS comments don't render in the mail either ways. My formatter is acting weird when I place Jinja2 comments inside the ``style`` tag

Reply:
Should I leave this as is or ignore the formatter?

Reply:
What formatter is this?

Reply:
Better Jinja v0.20